src/containers/forms/PlanForm/index.tsx

```diff
@@ -329,7 +329,7 @@ const PlanForm = (props: PlanFormProps) => {
                     </div>
                   ) : (
                     <div id="jurisdictions-select-container" className="mb-5">
-                      {values.jurisdictions.map((jurisdiction, index) => (
+                      {values.jurisdictions.map((_, index) => (
```

Comment:
These make more sense as `_` if they're the first param and not being referenced